include/Pattern.h

```diff
@@ -110,7 +110,9 @@ class LMMS_EXPORT Pattern : public TrackContentObject
 
 
 	TrackContentObjectView * createView( TrackView * _tv ) override;
-
+	
+	void setBGColor( QColor & c );
```

Comment:
Are these supposed to be here? I can't see these methods defined anywhere in Pattern.cpp, + they should be inherited from Track.h/Track.cpp.

Reply:
Probably not, I think I just forgot to move them to Track.h